src/pomodoro/TimerDigits.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type TimerDigitsProps = {
```

Comment:
Im not a fan of naming the `Props` type like `${ComponentName}Props`, especially if they are used only in one file because you can end up with unnecessary long names. But its your preference.

Reply:
You're right.